src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuiteImportExport.XmlPort.al

```diff
@@ -21,6 +23,33 @@ xmlport 149031 "AIT Test Suite Import/Export"
                 fieldattribute(Code; AITSuite.Code)
                 {
                     Occurrence = Required;
+
+                    trigger OnAfterAssignField()
+                    var
+                        AITTestSuiteRec: Record "AIT Test Suite";
+                        SameSuiteDifferentXMLErr: Label 'The test suite %1 is already imported with a different XML by the same app. Please delete the test suite and import again.', Comment = '%1 = Test Suite Code';
+                        SameSuiteDifferentAppErr: Label 'The test suite %1 is already imported by a different app. Please rename the test suite and import again.', Comment = '%1 = Test Suite Code';
+                    begin
+                        // Skip if the same suite is already imported by the same app
+                        // Error if the same suite is already imported with a different XML
+                        // Error if the same suite is already imported by a different app
+                        AITTestSuiteRec.SetLoadFields(Code, "Imported by AppId", "Imported XML's MD5");
```

Comment:
Minor - Same comment here, consider moving this to a local method it will increase the readability

Reply:
Won't fix: the code is very specific to this field and involves xmlport.skip(). Let's keep it as it is for now.